lib/beaker/options/parser.rb

```diff
@@ -224,7 +224,7 @@ def parse_args(args = ARGV)
       # @raise [ArgumentError] if a hosts file is generated, but it can't
       #   be read by the HostsFileParser
       def parse_hosts_options
-        if File.exists?(@options[:hosts_file])
+        if @options[:hosts_file].nil? || File.exists?(@options[:hosts_file])
```

Comment:
Should this be negated? `!@options[:hosts_file].nil?`

Otherwise it'll enter the `if` statement and `nil` will be passed to `HostsFileParser.parse_hosts_file`, which I'm assuming is bad.